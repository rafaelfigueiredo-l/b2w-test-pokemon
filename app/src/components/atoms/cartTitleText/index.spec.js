import CartText from "./";

describe("<CartText />", () => {
  const component = <CartText>Texto</CartText>;

  const mounted = mount(component);
  const wrapper = shallow(component);

  it("should mount correctly CartText component", () => {
    expect(wrapper.find(".cartText")).toHaveLength(1);
    expect(mounted).toMatchSnapshot();
  });

  it("should unmount correctly CartText component", () => {
    expect(mounted.unmount()).toEqual({});
  });

  it("should have a children", () => {
    expect(wrapper.find(".cartText").children()).toHaveLength(1);
  });
});
