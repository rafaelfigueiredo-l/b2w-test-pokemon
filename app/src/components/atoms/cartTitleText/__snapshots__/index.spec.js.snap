// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CartText /> should mount correctly CartText component 1`] = `
<styled.span
  className="cartText"
>
  <span
    className="sc-AxjAm eKjPIR cartText"
  >
    Texto
  </span>
</styled.span>
`;
